### DISCUSSION TITLE
Stand up Friday May 26th

### DISCUSSION THREAD

**Discussion by BrandonMagana** [Bugs-io/TC3001B#84] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
- TBH I didn't work on the project

**What are your priorities for the next 24 hours?**:
- Debug @CarlosCRG19 PR 

**Blockers**:
- None

**Shoutouts**:
- @CarlosCRG19 for working on the project

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Integrated dummy graphs in to the grid
- Worked graphs sizes issues and reponsiveness

**What are your priorities for the next 24 hours?**:
- Accomplished a user friendly responsiveness
- Set up some kind of middleware or interface for easy graph integration
- Work on Luis Gerardo en other docs during the weekend

**Blockers**:
- None

**Shoutouts**:
- @Jacobgtz024 for already pumping out graphs with Python

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Finish endpoints up to the 4th graph 

**What are your priorities for the next 24 hours?**:
- Work on radar graph
- Work on a function that returns an xlsx with the useless certs

**Blockers**:
- None

**Shoutouts**:
- @todos por guapos
